src/components/PostCommentForm/PostCommentForm.jsx

```diff
@@ -1,9 +1,73 @@
-const PostCommentForm = () => {
+import { useContext, useEffect } from "react";
+import { UserContext } from "../../contexts/User";
+import styles from "./PostCommentForm.module.css";
+import { useState } from "react";
+import { postComment } from "../../utils/apis";
+import CommentCard from "../CommentCard/CommentCard";
+
+const PostCommentForm = ({ article_id }) => {
+    const commentAuthor = useContext(UserContext).user;
+    const [newComment, setNewComment] = useState("");
+
+    const [addedComment, setAddedComment] = useState([]);
+
+    const handleNewComment = (e) => {
+        setNewComment(e.target.value);
+    };
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        if (newComment !== "") {
+            postComment(article_id, commentAuthor.username, newComment);
+            setAddedComment((curr) => {
```

Comment:
if we pass in the comments list array, and update the state here in exactly the same way, it'll save us from having to have this new state of addedcomments and then having to also render this in this component :)